app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_like_counter
+    post.increment!(:likes_counter)
+  end
+
+  private :update_like_counter
```

Comment:
- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_like_counter` method every time a new `like` has been saved to the database.